news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,225 @@
+
+The products' list displays on each line the product name, reference, stock location (if available), quantity, tax price and total price for the products. The total price follows the customer group configuration. If the customer group sets the prices with or without tax, it impacts the displayed amount in the total column and adds the related label "tax excluded" or "tax included".
+
+The list displays the 10 firsts products. The remaining products can be seen by expanding the list.
+
+There is a new hook called displayOrderPreview on the bottom of the preview.
+
+![Order preview's hook](/assets/images/2020/05/hook-preview.png "Order preview's hook")
+
+It allows modules to add new elements on the preview. For more specific needs, the preview can be changed by overriding template, find (here)[https://devdocs.prestashop.com/1.7/modules/concepts/templating/admin-views/] more information.
+
+
+
+
+- **Editable order status labels:** Allows to change the order status directly from the listing.
+ 
+![Edit a status](/assets/images/2020/05/editable_order_statuses_labels.png "Edit a status")
+
+***What is the goal***:
+
+It allows the merchant fast management of the order fulfilment.
+
+***How it works***: 
+
+When the user clicks then on the status label, it displays a drop-down list with all of the available statuses. By selecting one of the items on the list, the user can update the order status.
+
+
+- **Statuses' colour**: Color according to the order status
+
+![Order statuses by category](/assets/images/2020/05/order_status_labelsx2.png "Order statuses by category")
+
+***What is the goal***: 
+
+The colours used by default to contextualize the status is based on a wide range of colours without following any real logic. To bring a structure and meaning to the statuses colours, the statuses are grouped in 4 colours categories:
+
+- orders waiting for a customer action as in waiting for a bank check in dark blue
+- orders waiting for a merchant action following an accepted payment in light blue
+- when no action is needed from the merchant as when the product is delivered in green
+- special status as payment error order in red or black
+
+The merchants have a concise view of the work remaining to be done.
+
+***How it works***: 
+
+The statuses colour is set by default. They are still configurable in the Order Setting page.
+
+
+- **Link to the customer's profile**: Creates easy access to the customer's profile by clicking on its name.
+
+![Right click on the customer's link](/assets/images/2020/05/link-customer.png "Right click on the customer's link")
+
+***What is the goal***: 
+
+Before the merchant used to have to open the order to get access to the customer page. Now, the merchant can do it from the orders listing.
+
+***How it works***:
+
+Clicking on the customer name opens the customer page in a new tab.
+
+
```

Comment:
ttoine is going to telle you to reduce the space here
1 between 2 subparts
2 between main parts